<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Komlan Atitey</title>
    <link rel="shortcut icon" type="image/png" href="images/imageicon.png" />
    <link rel="stylesheet" href="style.css">


    <!-- Add icon library -->
    <script src="/js/mathjax-config.js"></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
        integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css"
        integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css"
        crossorigin="anonymous" title="hl-light"> 
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css"
        crossorigin="anonymous" title="hl-dark" disabled>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css"
        integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA=="
        crossorigin="anonymous">
    <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js"
        integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg=="
        crossorigin="anonymous" async></script>
    <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous"
        async></script>
    <link rel="stylesheet"
        href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
    <link rel="stylesheet" href="/css/wowchemy.css">
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-111897145-1"></script>
    <script>
        window.dataLayer = window.dataLayer || [];
        function gtag() {
            dataLayer.push(arguments);
        }
        function trackOutboundLink(url, target) {
            gtag('event', 'click', {
                'event_category': 'outbound',
                'event_label': url,
                'transport_type': 'beacon',
                'event_callback': function () {
                    if (target !== '_blank') {
                        document.location = url;
                    }
                }
            });
            console.debug("Outbound link clicked: " + url);
        }
        function onClickCallback(event) {
            if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
                return;
            }
            trackOutboundLink(event.target, event.target.getAttribute('target'));
        }
        gtag('js', new Date());
        gtag('config', 'UA-111897145-1', {});
        document.addEventListener('click', onClickCallback, false);
    </script>
    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
    <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Home">
    <link rel="manifest" href="/index.webmanifest">
    <link rel="icon" type="image/png"
        href="/images/icon_hufdd866d90d76849587aac6fbf27da1ac_464_32x32_fill_lanczos_center_2.png">
    <link rel="apple-touch-icon" type="image/png"
        href="/images/icon_hufdd866d90d76849587aac6fbf27da1ac_464_192x192_fill_lanczos_center_2.png">
    <link rel="canonical" href="/">
    <meta property="twitter:card" content="summary">
    <meta property="og:site_name" content="Home">
    <meta property="og:url" content="/">
    <meta property="og:title" content="Home">
    <meta property="og:description" content="Postdoctoral researcher">
    <meta property="og:image"
        content="/images/icon_hufdd866d90d76849587aac6fbf27da1ac_464_512x512_fill_lanczos_center_2.png">
    <meta property="twitter:image"
        content="/images/icon_hufdd866d90d76849587aac6fbf27da1ac_464_512x512_fill_lanczos_center_2.png">
    <meta property="og:locale" content="en-us">
    <meta property="og:updated_time" content="2022-11-21T18:08:38-04:00">
    <style>
        .topnav {
            overflow: hidden;
            background-color: #333;
        }

        .topnav a {
            float: left;
            color: #f2f2f2;
            text-align: center;
            padding: 14px 16px;
            text-decoration: none;
            font-size: 17px;
        }

        .topnav a:hover {
            background-color: #ddd;
            color: black;
        }

        .topnav a.active {
            background-color: #04AA6D;
            color: white;
        }

        figure {
            border: 0px #cccccc solid;
            padding: 20px;
            margin: auto;
        }

        figcaption {
            background-color: white;
            color: black;
            font-style: italic;
            padding: 2px;
            text-align: center;
        }

        header {
            display: flex;
            position: fixed;
            top: 0;
            left: 0;
            right: 0;
            height: 50px;
            line-height: 50px;
            background-color: #eee;
            width: 100%;
            z-index: 2;
        }

        header * {
            display: inline;
            height: 50px;
        }

        header ul {
            padding: 0;
        }

        header li {
            margin-left: 20px;
        }



        #hero .profile-img {
            width: 300px;
        }

        footer {
            text-align: center;
            padding: 50px;
        }

        div {
            /* border: 1px solid black;*/
            /* background-color: lightblue;*/
            padding-top: 30px;
            padding-right: 30px;
            padding-bottom: 30px;
            padding-left: 30px;
        }
        /* Create two equal columns that floats next to each other */
        .column {
            float: left;
            width: 60%;
            padding: 10px;
            height: 380px;
            /* Should be removed. Only for demonstration */
        }

        /* Clear floats after the columns */
        .row:after {
            content: "";
            display: table;
            clear: both;
        }
    </style>
</head>

<body>

<header id="header">
    <img src="images/komlan.png" class="profile-img">
    <nav id="nav-bar">
        <ul>
            <li><a href="#hero">Home</a></li>
            <li><a href="#about">Bio</a></li>
            <li><a href="manuscript/Resume_Atitey.pdf">CV</a></li>
            <li><a href="#research">Research Interests</a></li>
            <li><a href="#work">Publications</a></li>
            <li><a href="#tools">Computational Tools / Software</a></li>
            <li><a href="#future">Projects &amp; Ongoing Research</a></li>
            <li><a href="#awards">Awards, Honors, &amp; Fellowships</a></li>
            <li><a href="#lecturer">Teaching</a></li>
            <li><a href="#contact">Contact</a></li>
        </ul>
    </nav>
</header>

    
    <main>

        <div class="container">
            <section id="hero">
                <figure>
                    <img src="images/komlan.png" alt="">
                    <figcaption>
                        <h2>Komlan Atitey, PhD</h2><br>
                        <h3>Research Fellow</h3><br>
                        <font color="DARKOLIVEGREEN">
                            <p>
                            <h4> Biostatistics and Computational Biology Branch (BCBB)<br>
                                National Institute of Environmental Health Sciences (NIEHS)<br>
                                National Institutes of Health (NIH)</h4>
                            </p><br>
                        </font>
                        <font color="NOT DARKOLIVEGREEN">
                            <address>
                                111 Tw Alexander Dr<br>
                                David P Rall Building<br>
                                Research Triangle Park, NC 27709, United States<br>
                            </address>
                        </font>
                        <figcaption>
                </figure>
            </section><br>

            <section id="about">
                <div class="section-inner">
                    <h2>Bio</h2><br>
                    <p style="text-align:left;"> Dr. Komlan Atitey is a Research Fellow in the Biostatistics and
                          Computational Biology Branch at the <a  href="https://www.niehs.nih.gov/" target="_blank" rel="noopener">National 
                          Institute of Environmental Health Sciences (NIEHS)</a>, where he specializes in 
                           high-dimensional data analysis, cancer evolution modeling, and AI-driven biological inference. 
                           His work bridges machine learning, mechanistic and probabilistic modeling, and systems biology
                           to unravel the mechanisms underlying tumor progression, gene-gene interactions, and environmental
                           influences on disease. He has developed multiple impactful computational tools, including 
                        <strong>MMQE</strong>, 
                         <strong>DEGBOE</strong>, 
                         <strong>MIBCOVIS</strong>, 
                         <strong>GIBOOST</strong>, 
                         <strong>CancerTrace</strong>, 
                        and
                        <strong>magicSubclonal</strong>, 
                         a physics-informed AI framework that
                           integrates chemical master equation dynamics to detect driver-gene subclones and assign risk labels.
                           Through these contributions, Dr. Atitey continues to advance single-cell analytics, network inference,
                           and precision medicine</a>.</p><br>

                    <p style="text-align:left;">He received his PhD in Computational Biology from the <a
                            href="https://www.swansea.ac.uk/engineering/" target="_blank" rel="noopener">
                            College of Engineering</a> at Swansea University, United Kingdom, in August 2019. 
                            Prior to that, he earned a bachelor’s and a master’s degree in Physics from the Faculty 
                           of Sciences at the University of Lomé, Togo, in 2006 and 2007, respectively, and a 
                          second master’s degree in Information and Communication Engineering from
                          <a href="https://english.hrbeu.edu.cn/" target="_blank" rel="noopener">Harbin Engineering
                            University</a> in Harbin, Heilongjiang, China,
                          in 2015. In 2023, he further expanded his expertise by completing two certifications 
                           from online courses offered by <a href="https://statistics.fas.harvard.edu/data-science"
                            target="_blank" rel="noopener">Harvard University</a>
                          in Massachusetts, United States: one in Data Science - Machine Learning, and another 
                          in High-Dimensional Data Analysis. </p><br>

                    <p style="text-align:left;">Before joining NIEHS in November 2020, Dr. Atitey served as a Postdoctoral 
                        Research Associate at the <a href="https://www.unl.edu/" target="_blank" rel="noopener">University of
                            Nebraska-Lincoln</a> in Lincoln, NE, USA, where he worked on a Multiscale Multicellular Mathematical 
                    Model of Immunogenicity. Earlier in his career, following his first master’s degree in Physics, he spent 
                    five years teaching high school Physics in Togo. </p><br>
                    <h4>
                        <p style="color:blue"><i class="fas fa-download  pr-1 fa-fw"></i> Download my <a
                                href="manuscript/Resume_Atitey.pdf" target="_blank">CV</a></p>
                    </h4>
                </div>
            </section>
                            
            <section id="work">
                <div class="section-inner">
                    <h2>Research Interests</h2><br>
                    
                    <ul>
                        <li style="text-align:left;"><h3>Single-Cell and High-Dimensional Data Analysis</h3>
                            <p>Uncovering patterns and dynamics in complex biomedical datasets.</p></li><br>
                        <li style="text-align:left;"><h3>Cancer Evolution and Gene Regulation</h3>
                            <p>Identifying driver and modulator genes that shape tumor progression.</p></li><br> 
                        <li style="text-align:left;"><h3>Computational Modeling and AI</h3>
                            <p>Developing physics-informed, probabilistic, and machine learning frameworks.</p></li><br> 
                        <li style="text-align:left;"><h3>Network Biology and Systems Analysis</h3>
                            <p>Reconstructing gene-gene and cell-cell interaction networks.</p></li><br> 
                        <li style="text-align:left;"><h3>Translational and Precision Medicine Applications</h3>
                            <p>Integrating computational insights for disease understanding and therapeutic strategies.</p></li><br>                       
                    </ul>
                </div>
            </section>                 
                            
            <section id="work">
                <div class="section-inner">
                    <h2>Publications</h2><br>
                    <ul>
                        <li style="text-align:left;">
                            <p><a href="https://www.sciencedirect.com/science/article/pii/S2001037025004830"
                            target="_blank" rel="noopener">&ldquo;CancerTrace: Multi-stage single-cell analysis of networked cancer 
                                evolution for driver and modulator gene identification&rdquo;</a> is out. 
                                CancerTrace is a time-aware, single-cell RNA-seq framework that infers patient-specific,
                                causal driver–modulator relationships by reconstructing stage-resolved expression dynamics, 
                                enabling directed influence mapping and in-silico perturbation without requiring matched DNA
                                or large cohorts.</br></br></p>
                        </li>
                        
                        <li style="text-align:left;">
                            <p><a href="https://www.sciencedirect.com/science/article/pii/S2001037025004477"
                                   target="_blank" rel="noopener">&ldquo;Physics-informed AI with chemical master equation dynamics 
                                for driver-gene subclone detection and risk labeling&rdquo;</a> is out. 
                        magicSubclonal is a physics-informed framework that models driver-gene burst dynamics to identify
                        maximally separable rare subclones from bulk transcriptomes, linking these states to downstream genes
                        with controlled false discovery and delivering reproducible, clinically predictive subclone labels
                        that outperform existing mixture, clustering, and deconvolution methods.</br></br></p>
                         </li>
                        
                        <li style="text-align:left;">
                            <p><a href="https://academic.oup.com/bib/article/26/4/bbaf415/8239639"
                                    target="_blank" rel="noopener">&ldquo;Boosting data interpretation with GIBOOST to enhance visualization
                                of complex high-dimensional data&rdquo;</a> is out. 
                                GIBOOST integrates multiple high-dimensional single-cell datasets to enhance spatial data visualization and 
                                improve interpretability by optimally combining complementary dimensionality reduction methods.</br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://academic.oup.com/bib/article/25/1/bbad455/7477800"
                                    target="_blank" rel="noopener">&ldquo;Model-based evaluation of spatiotemporal data reduction methods 
                                with unknown ground truth through optimal visualization and interpretability metrics&rdquo;</a> is out. 
                                This work enhances the visualization and interpretability of high-dimensional data without relying on ground 
                                truth by integrating five robust metrics, including a novel time-ordered Markov-based structural metric, 
                                into a semi-supervised hierarchical Bayesian model.</br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://www.sciencedirect.com/science/article/pii/S1532046422002027"
                                    target="_blank" rel="noopener">&ldquo;DEGBOE: Discrete time Evolution modeling of
                                    Gene mutation through Bayesian inference using qualitative Observation of mutation
                                    Events&rdquo;</a> is out. This work enhances the understanding of gene-gene
                                interactions in the dynamics of gene mutations that drive cancer development and
                                progression.</br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://www.frontiersin.org/articles/10.3389/fmolb.2022.777390/full"
                                    target="_blank" rel="noopener">&ldquo;Mathematical Modeling of Proliferative Immune
                                    Response Initiated by Interactions Between Classical Antigen-Presenting Cells Under
                                    Joint Antagonistic IL-2 and IL-4 Signaling&rdquo;</a> is out. This work investigates
                                the combination of machine learning and mechanistic modeling to comprehend the
                                mechanisms of the dynamic interplay of lymphocyte proliferation with the complex
                                interaction effects of cytokines during an immune response to pathogen
                                invasion.</br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://www.frontiersin.org/articles/10.3389/fgene.2019.00549/full?ref=https://githubhelp.com"
                                    target="_blank" rel="noopener">&ldquo;Comprehensive Review of Models and Methods for
                                    Inferences in Bio-Chemical Reaction Networks&rdquo;</a> is out. This investigation
                                demonstrates that the most common models of biochemical reaction networks involve
                                differential equations, Markov processes, mass action kinetics, and state space
                                representations. Additionally, the most common tasks associated with these models are
                                parameter inference and model identification. </br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://iopscience.iop.org/article/10.1088/2057-1976/aaef5c/meta"
                                    target="_blank" rel="noopener">&ldquo;Inferring distributions from observed mRNA and
                                    protein copy counts in genetic circuits&rdquo;</a> is out. This work presents a
                                statistical method to infer distributions of mRNA and protein counts from observed data.
                                The main advantage of this method is that it does not require any prior assumptions or
                                knowledge about the underlying chemical reactions. </br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://www.sciencedirect.com/science/article/pii/S0303264718301072"
                                    target="_blank" rel="noopener">&ldquo;Elucidating effects of reaction rates on
                                    dynamics of the lac circuit in Escherichia coli&rdquo;</a> is out. This work
                                quantifies the uncertainty of not knowing the accurate reaction rates of biochemical
                                reactions and aims to understand how these reaction rates affect gene expression.
                                </br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://www.liebertpub.com/doi/full/10.1089/cmb.2018.0055" target="_blank"
                                    rel="noopener">&ldquo;Determining the Transcription Rates Yielding Steady-State
                                    Production of mRNA in the Lac Genetic Switch of Escherichia coli&rdquo;</a> is out.
                                This work elucidates the regulatory dynamics of gene expression activation and
                                inactivation by evaluating the transcription rates that yield steady-state mRNA
                                production in active and inactive states of the lac circuit, and the results of the
                                studies can be used in synthetic biology applications to understand the limits of
                                genetic synthesis. </br></br></p>
                        </li>
                        <li style="text-align:left;">
                            <p><a href="https://link.springer.com/chapter/10.1007/978-3-319-21978-3_33" target="_blank"
                                    rel="noopener">&ldquo;A Novel Prediction Algorithm in Gaussian-Mixture Probability
                                    Hypothesis Density Filter for Target Tracking&rdquo;</a> is out. This work provides
                                an improved Gaussian mixture probability hypothesis density filter for multiple target
                                tracking to maintain estimates of the targets' current states. </br></br></p>
                        </li>
                    </ul>
                </div>
            </section>


            <section id="tools">
                <div class="section-inner">
                    <h2>Computational Tools</h2><br>
                    <ul>
                        <li style="text-align:left;">
                            <p><a href="https://github.com/komlanAtitey/CancerTrace"target="_blank"
                                   rel="noopener">&ldquo;CancerTrace&rdquo;</a>. A time-aware single-cell framework that infers patient-specific cancer
                                driver genes and their modulators, reconstructing stage-resolved regulatory dynamics and causal driver-modulator 
                                influence networks.</li><br>         

                        <!-- Animation GIF -->
                        <div style="text-align:center; margin-top:20px;">
                        <img src="images/modulator_driver_heterogeneity.gif" alt="Modulators trigger driver genes, promoting tumor growth." style="max-width:100%; height:auto; border:1px solid #ccc;">
                        </div><br>
                        
                        <li style="text-align:left;">
                            <p><a href="https://github.com/komlanAtitey/magicSubclonal"target="_blank"
                                   rel="noopener">&ldquo;magicSubclonal&rdquo;</a>. A framework integrating stochastic 
                                driver-gene dynamics and gene expression to resolve rare subclones and assign clinical risk in cancer cohorts.</li><br>

                        <!-- Animation GIF -->
                        <div style="text-align:center; margin-top:20px;">
                        <img src="images/clonal_expansion_qt.gif" alt="Clonal Expansion Animation" style="max-width:100%; height:auto; border:1px solid #ccc;">
                        </div><br>
                        
                        <li style="text-align:left;">
                            <p><a href="https://github.com/NIEHS/GIBOOST"target="_blank"
                                   rel="noopener">&ldquo;GIBOOST&rdquo;</a>.An AI-driven framework 
                                integrating multiple dimensionality reduction methods to enhance clustering sensitivity and preserve biologically 
                                meaningful structures in high-dimensional single-cell data.</li><br>
                        <li style="text-align:left;">
                             <p><a href=https://github.com/NIEHS/MUBCOVID"target="_blank"
                                 rel="noopener">&ldquo;MIBCOVIS&rdquo;</a>. A Bayesian framework benchmarking and optimizing high-dimensional 
                                 data visualization and interpretability, integrating multiple metrics 
                                 to guide dynamic and spatial single-cell analyses.</li><br>
                        <li style="text-align:left;">
                            <p><a href=https://github.com/atiteyk2/Multiscale_Multicellular_Quantitative_Evaluator-MMQE-"target="_blank"
                                rel="noopener">&ldquo;MMQE(Multiscale Multicellular Quantitative Evaluator)&rdquo;</a>. A hybrid computational model 
                                capturing cytokine-lymphocyte interactions to predict immune response dynamics.</li><br>     
                                
                    </ul>
                </div>
            </section>

                                               
            <section id="future">
                <div class="section-inner">
                    <h2>Projects &amp; Ongoing Research; Fellowships</h2><br>
                    <ul>
                        <li style="text-align:left;"><h3>Cancer Evolution and Single-Cell Dynamics</h3>
                            <p>Developing computational frameworks to track tumor evolution, identify driver and modulator genes,
                                and map subclonal progression using single-cell multi-omics data.</p></li><br>
                        <li style="text-align:left;"> <h3>Physics-Informed AI for Gene and Subclone Analysis</h3>
                             <p>Integrating chemical master equation dynamics with AI to detect high-impact gene mutations,
                                 quantify risk, and model stochastic biological processes.</p></li><br> 
                        <li style="text-align:left;"><h3>Network and Systems Biology Modeling</h3>
                             <p>Inferring gene-gene and cell-cell interaction networks, reconstructing dynamic clonal hierarchies, 
                                 and studying functional regulatory pathways.</p></li><br> 
                        <li style="text-align:left;"><h3>Complex Image Integration and Analysis</h3>
                        <p>Designing computational pipelines for complex-valued biomedical image processing, enhancing feature extraction, 
                            visualization, and integration for cancer subtyping.</p></li><br> 
                        <li style="text-align:left;"><h3>Time-Varying Network Inference and Stochastic Systems</h3>
                        <p>Modeling gene interactions and clonal transitions over time using Markov processes, entropy-based scaling, 
                            and Bayesian frameworks.</p></li><br>  
                    </ul>
                </div>
            </section>

                            
            <section id="awards">
                <div class="section-inner">
                    <h2>Awards, Honors, &amp; Fellowships</h2><br>
                    <ul>
                        <li style="text-align:left;"><h3>Quantitative Approaches in Spatial Multi-Omics for Guiding Personalized Medicine | June 2025</h3>
                            <p>Event hosted by Biomedical Data Science Innovation Lab (BDSIL), Nashville, Tennessee</p></li><br>
                        <li style="text-align:left;"> <h3>Codeathon: Clinical and Single-Cell Transcriptomics for Pneumonia | Sep 2023</h3>
                             <p>Event hosted by NIAID, CZI, BV-BRC, and Northwestern University in Chicago</p></li><br> 
                        <li style="text-align:left;"><h3>Award for PAIR-UP UNC Imaging Workshop | Jan 2023</h3>
                             <p>Workshop hosted by the University of North Carolina at Chapel Hill in North Carolina.<br>
                             Training in advanced microscopy for imaging live cells using transmitted light, fluorescent probes, and fluorescent
                                 biosensors.</p></li><br> 
                        <li style="text-align:left;"><h3>Award for Research Excellence (FARE 2022), USA | Jul 2022</h3>
                        <p>NIH Awarded Study Section: Informatics / Computational Biology / Systems Biology<br>
                         <a href="https://www.niehs.nih.gov/careers/research/fellows/involvement/awards/excellence/index.cfm" 
                             target="_blank" rel="noopener">View Award Details</a></p></li><br> 
                        <li style="text-align:left;"><h3>United Kingdom Highly Prestigious Zienkiewcz PhD Fully Funded Scholarship | Feb 2016</h3>
                        <p>Scholarship for PhD research in Engineering, Swansea University, United Kingdom.<br>
                         Reserved for outstanding students with top academic performance (First Class Honors Degree).</p></li><br>  
        
                        <li style="text-align:left;"><h3>China Scholarship Council (CSC) - Full Master's Scholarship | Mar 2012</h3>
                     <p>Awarded a fully funded CSC Scholarship for a Master’s degree in Engineering at Harbin Engineering University, Heilongjiang, China.</p></li><br>                        
                    </ul>
                </div>
            </section>

                            
            <!--section height-->
            <section id="lecturer">
                <div class="section-inner">
                    <h2>Teaching: Biostatistics and Bioinformatics Short Courses</h2><br>

                    <li><i class="fas fa-download  pr-1 fa-fw"></i> See the <a href="manuscript/course1.pdf"
                            target="_blank"> course for module 5, 6, and 7 </a></p>
                    </li><br><br>
                    <li><i class="fas fa-download  pr-1 fa-fw"></i> See the <a href="manuscript/course2.pdf"
                            target="_blank"> course for module 8, 9, and 10 </a></p>
                    </li><br>
                </div>
            </section>


            <section id="contact">
                <div class="wrapper">
                    <h2>Contact me</h2><br>
                    <p>You can find me on:</p><br>
                    <ul class="network-icon" aria-hidden="true">
                        <a href="https://www.facebook.com/atitey.komlan" target="_blank" rel="noopener"><i
                                class="fab fa-facebook big-icon"></i></a><br>
                        <a href="https://twitter.com/KomlanAtitey" target="_blank" rel="noopener"><i
                                class="fab fa-twitter big-icon"></i></a><br>
                        <a href="https://scholar.google.com/scholar?hl=en&as_sdt=0%2C34&q=Komlan+Atitey&oq=Atite"
                            target="_blank" rel="noopener"><i class="fas fa-graduation-cap big-icon"></i></a><br>
                        <a href="https://github.com/KomlanAtitey62" target="_blank" rel="noopener"><i
                                class="fab fa-github big-icon"></i></a><br>
                        <a href="https://github.com/komlanAtitey" target="_blank" rel="noopener"><i
                                class="fab fa-github big-icon"></i></a><br>
                        <a href="https://www.linkedin.com/in/komlan-atitey-phd-34082998/" target="_blank"
                            rel="noopener"><i class="fab fa-linkedin big-icon"></i></a><br>
                    </ul><br>
                    <p>komlan.atitey@nih.gov / atiteydavidkomlan@yahoo.fr</p>
                </div>
            </section>
        </div>
        <div class="footer">
            © Copyright eglikokotako, 2023
        </div>
    </main>
</body>
</html>
